rfc6962-bis.xml

```diff
@@ -1321,7 +1321,10 @@ but it is expected there will be a variety.
 	  </t>
 	  <t hangText="Version">
 	    The version of the protocol supported by the log (currently 1 or 2).
-	  </t>
+    </t>
+    <t hangText="Maximum Chain Length">
+      The longest chain submission the log is willing to accept, if the log chose to limit it.
+    </t>
```

Comment:
Fixed indentation.
I don't think we need to say it's in decimal - the metadata format is not strictly specified since how it is distributed is client-specific.